packages/dart_locations_repository_interface/lib/src/locations_service.dart

```diff
@@ -11,7 +11,7 @@ class LocationsService {
 
   final LocationsRepositoryInterface _locationsRepositoryInterface;
 
-  /// Checks if the internal interface uses the location 
+  /// Checks if the internal interface uses the location
```

Comment:
I am not sure I would make this grand change right now in the project, it might cause issues later on